python/.python-version

```diff
@@ -1,3 +1,3 @@
-3.6.10
-3.7.6
 3.8.2
+3.7.6
+3.6.10
```

Comment:
Does this change default python version selection?

Reply:
sorry, missed this question: yes, the first one in the list is the default version when you cd into the directory. There are more than one version listed so that `tox` can find them and use them